frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -17,20 +17,20 @@
     <td style="width: 100%" class="contentrow">
       <router-link
         :to="routerLink"
-        class="text-decoration-none text-decoration-hover-underline black--text"
+        class="text-decoration-none text-decoration-hover-underline black--text font-weight-medium d-block"
       >
         {{ title }}<br />
       </router-link>
       <span class="e-subtitle">{{ location }}</span>
     </td>
     <td class="contentrow avatarrow overflow-visible">
```

Comment:
This was already the case and I would say this is an edge case. I assume there are most of the time a maximum of 4 user assigned as responsible. We could alleviate this with groups if this is often the case instead of trying to fit more bubbles on the screen. E.g "Wolfstufe" or "Gruppe 1"

Reply:
Ah, I might have not been clear: I'm was more pointing to the one **without** any responsibles: There's no reason why the title needs to be cut (there's enough space).

Reply:
Sorry. Since this is currently a table, this is the expected behavior.

Reply:
I removed the whitespace nowrap. It should now wrap as normal (also the max-width of the avatarcell is reduced)

Reply:
<img width="304" alt="Bildschirmfoto 2022-12-31 um 11 07 34" src="https://user-images.githubusercontent.com/3001985/210132814-70cf985e-f637-43a9-afe2-0534949e2db2.png">